src/Compilers/VisualBasic/Portable/Errors/Errors.vb

```diff
@@ -1968,7 +1973,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         IDS_LogoLine1 = 56007
         IDS_LogoLine2 = 56008
         IDS_VBCHelp = 56009
-        IDS_InvalidPreprocessorConstantType = 56010
+        ' IDS_InvalidPreprocessorConstantType = 56010           Replaced by ERR_InvalidPreprocessorConstantType
```

Comment:
From what I can see in the rest of the file, old values that are no longer used are commented out, with a note about why they were so, which leads to using new enum values for new messages.
Do you see otherwise? should this line be removed and the number marked as available so that it can be reused?

---
In reply to: [105784631](https://github.com/dotnet/roslyn/pull/17113#discussion_r105784631) [](ancestors = 105784631)